oss-fuzz/arvo_utils.py

```diff
@@ -0,0 +1,742 @@
+"""ARVO utilities module.
+
+This module provides utility functions for ARVO reproducer including:
+- Command execution functions
+- Version control operations
+- Docker operations
+- File system utilities
+- Dockerfile modification tools
+"""
+
+import logging
+import re
+import shutil
+import subprocess
+import tempfile
+import warnings
+from datetime import datetime
+from pathlib import Path
+from typing import Any, Dict, List, Optional, Tuple, Union
+
+import pytz
+
+# Global constants
+OSS_OUT = OSS_WORK = OSS_ERR = Path("/tmp")
+
+# Configure logging
+logging.basicConfig(level=logging.INFO,
+                    format='%(asctime)s - %(levelname)s - %(message)s')
+
+# Suppress Google auth warnings
+warnings.filterwarnings("ignore",
+                        category=UserWarning,
+                        module="google.auth._default")
+
+
+def info(message: str) -> None:
```

Comment:
can we just use `logging.info`, `logging.warning` etc directly instead of through these functions?